docassemble/MACourts/data/sources/housing_courts_addcode+add4new.json

```diff
@@ -0,0 +1,521 @@
+    "name": "Western Housing Court - Springfield Session",
+    "phone": "(413) 748-7838",
+    "has_po_box": true,
+    "location": {
+      "latitude": 42.100362,
+      "longitude": -72.588744
+    },
+    "address": {
+      "city": "Springfield",
+      "zip": "01102",
+      "orig_address": "37 Elm Street, P.O. Box 559, Springfield, MA 01102",
+      "county": "Hampden County",
+      "state": "MA",
+      "address": "37 Elm Street, P.O. Box 559"
+    },
+    "description": "The Western Housing Court in Springfield serves Agawam, Blandford, Brimfield, Chester, Chicopee, East Longmeadow, Granville, Hampden, Holland, Holyoke, Longmeadow, Ludlow, Monson, Montgomery, Palmer, Russell, Southwick, Springfield, Tolland, Wales, West Springfield, Westfield and Wilbraham."
+  },
+  {
+    "fax": "",
+    "code": "H84",
+    "name": "Eastern Housing Court - Middlesex Session",
+    "phone": "(781) 306-2715",
+    "has_po_box": false,
+    "location": {
+      "latitude": 42.4048336,
+      "longitude": -71.0893853
+    },
+    "address": {
+      "city": "Medford",
+      "address": "4040 Mystic Valley Parkway",
+      "state": "MA",
+      "zip": "02155",
+      "county": "Middlesex County",
+      "orig_address": "4040 Mystic Valley Parkway, Medford, MA 02155"
+    },
+    "description": "The Middlesex Session of the Eastern Housing Court serves Arlington, Belmont, and Cambridge, Medford and Somerville"
+  },
+  {
+    "fax": "",
+    "code": "H77",
+    "name": "Northeast Housing Court - Woburn Session",
+    "phone": "(978) 689-7833",
+    "has_po_box": false,
+    "location": {
+      "latitude": 42.500543,
+      "longitude": -71.1656604
+    },
+    "address": {
+      "city": "Woburn",
+      "address": "200 Trade Center",
+      "unit": "Courtroom 540 - 5th Floor",
+      "state": "MA",
+      "zip": "01801",
+      "county": "Middlesex County",
+      "orig_address": "200 Trade Center, Courtroom 540 - 5th Floor, Woburn, MA 01801"
+    },
+    "description": "The Woburn session of the Northeast Housing Court serves Bedford, Burlington, Concord, Everett,Lexington, Lincoln, Malden, Melrose, North Reading, Reading, Stoneham, Wakefield, Waltham, Watertown, Weston, Wilmington, Winchester, and Woburn."
+  },
+  {
+    "fax": "(508)362-0290",
+    "code": "H83",
+    "name": "Southeast Housing Court - Barnstable session",
+    "phone": "(508)375-6967",
+    "has_po_box": true,
+    "location": {
+      "latitude": 41.735149,
+      "longitude": 41.735149
+    },
+    "address": {
+      "city": "Boston",
+      "zip": "02630",
+      "orig_address": "3195 Main Street, Barnstable, MA 02630",
+      "county": "barstable",
+      "state": "MA",
+      "address": "Barnstable, MA"
+    },
+    "description": "The barnstable division of the Housing court is located in Barnstable, MA"
+  },
+  {
+    "fax": "",
+    "code": "H84",
+    "name": "Eastern Housing Court - Chelsea Session",
+    "phone": "(617)788-8485",
+    "has_po_box": false,
```

Comment:
can we add address (from Chelsea District Court) and description from this page: https://www.mass.gov/locations/eastern-housing-court-chelsea-session